driving game/main.js

```diff
@@ -21,4 +23,16 @@ document.addEventListener('keydown', function (e) {
     $car.className = 'east';
     carDirection.currentDirection = 'east';
   }
+  if (e.key === ' ') {
+    intervalId = setInterval(startCar, 16)
+  }
 });
+
+var positionX = 0;
+
+var intervalId = null;
+
+function startCar () {
+  positionX += 5;
+  $car.style.left = positionX + 'px';
+}
```

Comment:
Why have a postionX variable when you have an x property in your data model?
`carDirection.x+=5`

Reply:
Yea that makes total sense. I think I'm just over complicating it.